pkg/rpc/server/server.go

```diff
@@ -988,7 +987,13 @@ func (s *Server) invokeFunction(reqParams request.Params) (interface{}, *respons
 	if responseErr != nil {
 		return nil, responseErr
 	}
-	tx := &transaction.Transaction{}
+	method, err := reqParams.ValueWithType(1, request.StringT).GetString()
+	if err != nil {
+		return nil, response.ErrInvalidParams
+	}
+	tx := &transaction.Transaction{
+		Script: []byte{1},
```

Comment:
?

Reply:
`(c *Client) AddNetworkFee` invokes `verify` method to determine network fee for deployed contracts [here](https://github.com/nspcc-dev/neo-go/blob/master/pkg/rpc/client/rpc.go#L565). We have a [test](https://github.com/nspcc-dev/neo-go/blob/master/pkg/rpc/server/client_test.go#L147) for it which uses [verification_contract.go](https://github.com/nspcc-dev/neo-go/blob/master/pkg/rpc/server/testdata/verification_contract.go). This contract has `verify` method and invokes `runtime.GetScriptContainer()` syscall which fails during `invokefunction` because of an empty script of this dummy transaction. The similar situation may occure with some real contract.

So this `Script: []byte{1},` is added in order to be able to serialize the script-container (which is our transaction with signers) in case if someone will need it. Although it is not very fair.